relay-server/src/services/processor/span/processing.rs

```diff
@@ -20,24 +23,50 @@ use crate::services::outcome::{DiscardReason, Outcome};
 use crate::services::processor::{ProcessEnvelopeState, ProcessingError};
 use crate::utils::ItemAction;
 
-pub fn process(state: &mut ProcessEnvelopeState, config: Arc<Config>) {
+fn get_normalize_span_config<'a>(
+    config: Arc<Config>,
+    received_at: DateTime<Utc>,
+    global_measurements_config: Option<&'a MeasurementsConfig>,
+    project_measurements_config: Option<&'a MeasurementsConfig>,
+) -> NormalizeSpanConfig<'a> {
+    let transaction_aggregator_config =
+        AggregatorConfig::from(config.aggregator_config_for(MetricNamespace::Transactions));
```

Comment:
I probably messed this up initially:

For transactions, this normalization step ensures that the payload aligns with the limits of the metrics aggregator. Span metrics potentially go into a different aggregator, so we should align with that.

```suggestion
    let aggregator_config =
        AggregatorConfig::from(config.aggregator_config_for(MetricNamespace::Spans));
```